### ISSUE TITLE
Is there any chance to get a colorful sidebar also in Sierra Mail app?

### ISSUE THREAD

**Issue by Canubiz** [w0lfschild/colorfulSidebar_9#4]
I am aware colorfulSidebar9 is supposed to just give the Finder colored icons in the sidebar – which it is great at! But would be so cool to also get back colored icons in the Mail app's sidebar. :)

**Comment by w0lfschild** [comment]
Maybe if I have the time I can look into it but chances are low right now.

**Comment by w0lfschild** [comment]
So I finally went ahead and did this... 3 years later. Probably have an update out in the coming week or so hopefully.

![Screen Shot 2020-03-23 at 10 52 50 PM](https://user-images.githubusercontent.com/1920148/77392884-2ba56d00-6d59-11ea-85df-7795251b93e2.png)

**Comment by Canubiz** [comment]
Sorry for reopening this thread – but it seems colorful sidebar in Mail only works with dark mode. In light mode Mail still has plain, gray icons. 
Any chance this could be an easy addition?

Btw, thanks a lot for adding this feature after all that time. Good things take time to ripe and at least in Dark Mode Mail works much better for me now. :) (I just recently upgraded to a new Mac with Catalina, immediately installed MacForge and bought colorfulSidebarX when I saw you added this. :))
